src/gc/gc.cpp

```diff
@@ -11373,6 +11374,13 @@ void gc_heap::adjust_limit_clr (uint8_t* start, size_t limit_size,
         }
         acontext->alloc_ptr = start;
     }
+    else  
+    {  
+        // If the next alloc context is right up against the current one it means we are absorbing the min  
+        // object, so need to account for that.  
+        acontext->alloc_bytes += (start - acontext->alloc_limit);  
```

Comment:
LOH ac's are tracked differently - for each large object allocation we just have a local ac on the stack and the size of the ac will just be what you asked for. so all we need to do is adding that number of bytes to alloc_bytes_loh.